src/v0/destinations/attentive_tag/util.js

```diff
@@ -14,12 +14,19 @@ const { mappingConfig, ConfigCategory } = require('./config');
  * The keys should not contain any of the values inside the validationArray.
  * STEP 1: Storing keys in the array.
  * Checking for the non-valid characters inside the keys of properties.
+ * ref: https://docs.attentivemobile.com/openapi/reference/tag/Custom-Events/
  * @param {*} payload
  * @returns
  */
-const getPropertiesKeyValidation = (payload) => {
+const arePropertiesValid = (properties) => {
+  if (!isDefinedAndNotNullAndNotEmpty(properties)) {
+    return true;
```

Comment:
should this be `false` ?

Reply:
no, it will be true because properties is not a mandatory field.